worlds/stardew_valley/options.py

```diff
@@ -533,6 +554,46 @@ class FriendsanityHeartSize(Range):
     # step = 1
 
 
+class Booksanity(Choice):
+    """Shuffle Books?
+    None: All books behave like vanilla
+    Power: Power books are turned into checks
+    Power and Skill: Power and skill books are turned into checks.
+    All: Lost books are also included in the shuffling
+    """
```

Comment:
Make sure this will look how you want it to look on webhost.

Reply:
What part makes you worried?

Reply:
The `"""Shuffle Books?` line. On the newer version of the webhost, I think if you keep it like that, it'll tab subsequent lines for some reason.

Here's what your ginger island option currently looks like on the beta site:
![image](https://github.com/ArchipelagoMW/Archipelago/assets/24858499/4b53574e-f8ee-48c4-afd6-89ab3e40ef31)

Just, I can see someone actually wanting this, so I don't want to just assume it's the wrong way to write it, just I figure most probably don't like it like this.

Reply:
I feel like that's okay, at least a tiny bit more readable than if it was untabbed. I'll keep it